### DISCUSSION TITLE
Phalcon 5 doesn't work on PHP 8

### DISCUSSION THREAD

**Discussion by PHProger-themus** [phalcon/phalcon#239] [General]
Hello, I've installed Phalcon 5.0.0beta3 on PHP 8.0.8 Thread Safe x64 (Windows OC) from [this](https://github.com/phalcon/cphalcon/releases/tag/v5.0.0beta3) page (the last one), copied extension to php's extension folder and added extension to php.ini, psr extension was also added.
I see that both PSR and Phalcon are loaded in phpinfo(), that means extension works fine, but when I cloned [this](https://github.com/phalcon/tutorial) project to see Phalcon's work, it gave me an error:

`Fatal error: Uncaught Error: Class "Phalcon\Loader" not found in D:\web\www\phalcon-website.ru\public\index.php:14 Stack trace: #0 {main} thrown in D:\web\www\phalcon-website.ru\public\index.php on line 14`

Seems like I forgot something, extension is working but there's errors. What's the problem?

**Comment by Jeckerson** [comment]
Hello, please check changelog for any errors - https://github.com/phalcon/cphalcon/blob/master/CHANGELOG-5.0.md

**Comment by borisdelev** [comment]
Try \Phalcon\Loader\Loader
... See the changelog, Jeckerson link provide
